src/modules/px4iofirmware/protocol.h

```diff
@@ -218,113 +187,42 @@ enum {							/* DSM bind states */
-
-#define PX4IO_P_SETUP_THERMAL			26	/**< thermal management */
-
-#define PX4IO_P_SETUP_AIRMODE 			27 	/**< air-mode */
-
-#define PX4IO_P_SETUP_ENABLE_FLIGHTTERMINATION	28	/**< flight termination; false if the circuit breaker (CBRK_FLIGHTTERM) is set */
+#define PX4IO_P_SETUP_SBUS_RATE			16	/**< frame rate of SBUS1 output in Hz */
+#define PX4IO_P_SETUP_THERMAL			17	/**< thermal management */
+#define PX4IO_P_SETUP_ENABLE_FLIGHTTERMINATION	18	/**< flight termination; false if the circuit breaker (CBRK_FLIGHTTERM) is set */
 
 #define PX4IO_THERMAL_IGNORE			UINT16_MAX
 #define PX4IO_THERMAL_OFF			0
 #define PX4IO_THERMAL_FULL			10000
 
-/* autopilot control values, -10000..10000 */
-#define PX4IO_PAGE_CONTROLS			51	/**< actuator control groups, one after the other, 8 wide */
-#define PX4IO_P_CONTROLS_GROUP_0		(PX4IO_PROTOCOL_MAX_CONTROL_COUNT * 0)	/**< 0..PX4IO_PROTOCOL_MAX_CONTROL_COUNT - 1 */
-#define PX4IO_P_CONTROLS_GROUP_1		(PX4IO_PROTOCOL_MAX_CONTROL_COUNT * 1)	/**< 0..PX4IO_PROTOCOL_MAX_CONTROL_COUNT - 1 */
-#define PX4IO_P_CONTROLS_GROUP_2		(PX4IO_PROTOCOL_MAX_CONTROL_COUNT * 2)	/**< 0..PX4IO_PROTOCOL_MAX_CONTROL_COUNT - 1 */
-#define PX4IO_P_CONTROLS_GROUP_3		(PX4IO_PROTOCOL_MAX_CONTROL_COUNT * 3)	/**< 0..PX4IO_PROTOCOL_MAX_CONTROL_COUNT - 1 */
-
-#define PX4IO_P_CONTROLS_GROUP_VALID		64
-#define PX4IO_P_CONTROLS_GROUP_VALID_GROUP0	(1 << 0) /**< group 0 is valid / received */
-#define PX4IO_P_CONTROLS_GROUP_VALID_GROUP1	(1 << 1) /**< group 1 is valid / received */
-#define PX4IO_P_CONTROLS_GROUP_VALID_GROUP2	(1 << 2) /**< group 2 is valid / received */
-#define PX4IO_P_CONTROLS_GROUP_VALID_GROUP3	(1 << 3) /**< group 3 is valid / received */
-
-/* raw text load to the mixer parser - ignores offset */
-#define PX4IO_PAGE_MIXERLOAD			52
-
 /* R/C channel config */
 #define PX4IO_PAGE_RC_CONFIG			53		/**< R/C input configuration */
-#define PX4IO_P_RC_CONFIG_MIN			0		/**< lowest input value */
-#define PX4IO_P_RC_CONFIG_CENTER		1		/**< center input value */
-#define PX4IO_P_RC_CONFIG_MAX			2		/**< highest input value */
-#define PX4IO_P_RC_CONFIG_DEADZONE		3		/**< band around center that is ignored */
-#define PX4IO_P_RC_CONFIG_ASSIGNMENT		4		/**< mapped input value */
-#define PX4IO_P_RC_CONFIG_ASSIGNMENT_MODESWITCH	100		/**< magic value for mode switch */
-#define PX4IO_P_RC_CONFIG_OPTIONS		5		/**< channel options bitmask */
+#define PX4IO_P_RC_CONFIG_ASSIGNMENT		0		/**< mapped input value */
+#define PX4IO_P_RC_CONFIG_ASSIGNMENT_KILLSWITCH	100	/**< magic value for kill switch */
+#define PX4IO_P_RC_CONFIG_OPTIONS		1		/**< channel options bitmask */
 #define PX4IO_P_RC_CONFIG_OPTIONS_ENABLED	(1 << 0)
 #define PX4IO_P_RC_CONFIG_OPTIONS_REVERSE	(1 << 1)
-#define PX4IO_P_RC_CONFIG_STRIDE		6		/**< spacing between channel config data */
+#define PX4IO_P_RC_CONFIG_STRIDE		2		/**< spacing between channel config data */
 
 /* PWM output - overrides mixer */
 #define PX4IO_PAGE_DIRECT_PWM			54		/**< 0..CONFIG_ACTUATOR_COUNT-1 */
```

Comment:
Old comment above. Maybe do a search for all mention of mixer to remove some of these.

Reply:
Done.